autospec/autospec.py

```diff
@@ -320,6 +321,10 @@ def package(args, url, name, archives, workingdir, infile_dict):
                                                tarball.name,
                                                tarball.version,
                                                tarball.release)
+        if filemanager.is_avx_candidate(mock_chroot):
+            print("Autospec found files candidates to AVX technology !")
+            print("Is recomended to set use_avx2 = true in options.conf")
```

Comment:
Poor grammar again, maybe "Set use_avx2 to 'true' in options.conf to use features"

Also, "recomended" is spelled wrong.